packages/daemon/src/service/plugin.ts

```diff
@@ -118,6 +130,31 @@ export class PluginManager {
       throw err;
     }
   }
+  async startInstall(pkg: PluginPackage, pyIndex?: string, force?: boolean): Promise<PluginInstallingResp> {
```

Comment:
Add comments for parameters, please.

Reply:
And the naming of this function looks weird, because "start" and "install" are verbs :(

Reply:
installAsync()?

Reply:
Why not just `install()`?

Reply:
`install` is used when we need to install a plugin synchronously.